turbo.json

```
@@ -23,7 +23,8 @@
       "outputs": ["dist/**"]
     },
     "build:storybook": {
-      "outputs": ["storybook-static/**"]
+      "outputs": ["storybook-static/**"],
+      "dependsOn": ["^build"]
```

ici je mettrais 
```suggestion
      "dependsOn": ["build"]
```

ce qui devrait corriger le problème

---

plus d'actualité après la suppression de turbo